<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../ressources/ressources.qrc" >:/images/images/tv.png</iconset>
  </property>
  <property name="dockNestingEnabled" >
   <bool>true</bool>
  </property>
  <property name="dockOptions" >
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QPushButton" name="aujourdhui" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>&amp;Maintenant</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="soiree" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>&amp;Soirée</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QToolButton" name="boutonJourAvant" >
            <property name="text" >
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelDate" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font" >
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>TextLabel</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="boutonJourApres" >
            <property name="text" >
             <string>></string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsViewProgrammes" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dockSoiree" >
   <property name="windowTitle" >
    <string>Soirée</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2" >
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item row="0" column="0" >
      <widget class="QListWidget" name="listeSoiree" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockVisu" >
   <property name="windowTitle" >
    <string>Visu</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3" >
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>9</number>
     </property>
     <item row="0" column="0" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QWidget" native="1" name="visu" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QToolButton" name="stop" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset resource="../ressources/ressources.qrc" >:/images/images/stop.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="record" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset resource="../ressources/ressources.qrc" >:/images/images/record.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelLogo" >
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockMaintenant" >
   <property name="windowTitle" >
    <string>Maintenant</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContentsMaintenant" >
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item row="0" column="0" >
      <widget class="ListMaintenant" name="listeMaintenant" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockDesc" >
   <property name="features" >
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle" >
    <string>Description</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <property name="horizontalSpacing" >
      <number>0</number>
     </property>
     <property name="verticalSpacing" >
      <number>0</number>
     </property>
     <item row="0" column="0" >
      <widget class="QTextBrowser" name="desc" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier" >
    <property name="title" >
     <string>&amp;Fichier</string>
    </property>
    <addaction name="separator" />
    <addaction name="action_Quitter" />
   </widget>
   <widget class="QMenu" name="menu_Edition" >
    <property name="title" >
     <string>&amp;Edition</string>
    </property>
    <addaction name="action_Programmes" />
    <addaction name="action_Configurer" />
    <addaction name="action_Canaux" />
   </widget>
   <widget class="QMenu" name="menu_Affichage" >
    <property name="title" >
     <string>&amp;Affichage</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuA_ide" >
    <property name="title" >
     <string>A&amp;ide</string>
    </property>
    <addaction name="actionA_propos" />
    <addaction name="actionA_propos_de_Qt" />
   </widget>
   <addaction name="menu_Fichier" />
   <addaction name="menu_Edition" />
   <addaction name="menu_Affichage" />
   <addaction name="menuA_ide" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="action_Quitter" >
   <property name="text" >
    <string>&amp;Quitter</string>
   </property>
  </action>
  <action name="action_Configurer" >
   <property name="text" >
    <string>&amp;Configurer...</string>
   </property>
  </action>
  <action name="action_Programmes" >
   <property name="text" >
    <string>&amp;Programmes...</string>
   </property>
  </action>
  <action name="actionA_propos" >
   <property name="text" >
    <string>A propos de l'application...</string>
   </property>
  </action>
  <action name="actionA_propos_de_Qt" >
   <property name="text" >
    <string>A propos de Qt...</string>
   </property>
  </action>
  <action name="action_Canaux" >
   <property name="text" >
    <string>C&amp;anaux...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ListMaintenant</class>
   <extends>QListWidget</extends>
   <header>listmaintenant.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ressources/ressources.qrc" />
 </resources>
 <connections/>
</ui>
